tensorflow/core/graph/mkl_graph_util.h

```diff
@@ -122,6 +125,23 @@ inline string GetMklEagerOpName(const string& name) {
   return string(kMklEagerOpPrefix) + name;
 }
 
+#ifdef ENABLE_INTEL_MKL_BFLOAT16
+static inline bool CheckBfloat16Support(DataType T) {
+  static absl::once_flag cpu_bfloat16_warn_once_flag;
+  // Restrict bfloat16 ops to platforms with at least AVX512 support, fall back
+  // to Eigen implementation.
```

Comment:
Nit: The explanation is incomplete.
```suggestion
  // to Eigen implementation otherwise.
```

Reply:
Done.
Question: I would have loved to click on the commit suggestion button in (github feature), but remember Google had clang issues because of automated bot as one of the checkins, so changed it manually. Are you aware if those problems still exist or have gone away, so that I can use it for future?

Reply:
I'm not sure. We can try using the accept suggestion feature for C++ changes so we'll see if it will work.

The github `Ubuntu Sanity` check only checks for python formatting errors. Incorrectly-formatted C++ changes should still pass the Sanity check and won't prevent us from pulling the PR in. Once it is pulled inside, the changes are ran through `clang-format` again anyway. I think it should work.